### ISSUE TITLE
Usability Testing for High Fidelity Prototypes 

### ISSUE THREAD

**Issue by lrchang2** [hackforla/access-the-data#102]
### Overview
We need to conduct usability testing of the high fidelity prototypes of the desktop and mobile version to see if the design is user friendly. 

### Action Items

- [x] Develop documentation for usability testing
- [x] Conduct 2 usability tests (one for desktop, one for mobile) using the Script and put notes in documentation
- [x] Put noteworthy observations about each test into FigJam Affinity Map under Usability Testing V2
- [x] review results and determine edits as a team during 1/12/22 team meeting
- [ ] develop next steps

### Resources/Instructions
[Usability Test Script](https://docs.google.com/document/d/1YJ-n7IU4eqJsw6_7-myNX0sKSWfWdW5AGJgHlddDbDM/edit)
[Version 2 of Usability Testing Notes (tasks are the same as usability testing V1 for low/mid fi) ](https://docs.google.com/spreadsheets/d/1LSn9gwVv1VDBNDIgepzHkhBxr-W57RyVXv-jb8u6vDQ/edit#gid=984765511)

[Figma for High Fi Wireframes (Mobile)](https://www.figma.com/proto/nS5IqnutUT2FuPwwJObHaY/Access-the-Data-Main-Figma?node-id=2%3A14&scaling=scale-down&starting-point-node-id=1769%3A4759&show-proto-sidebar=1)
[Figma for High Fi Wireframes (Desktop)](https://www.figma.com/proto/nS5IqnutUT2FuPwwJObHaY/Access-the-Data-Main-Figma?node-id=1911%3A5394&scaling=scale-down&page-id=2%3A16&starting-point-node-id=1911%3A5394&show-proto-sidebar=1)

[Figjam affinity map for usability results ](https://www.figma.com/file/doYnKXIUX9PC0XF7UcyU4P/Mid-fi-Wireframes-Usability-test-debrief-and-synthesis?node-id=16%3A221)

**Comment by lrchang2** [comment]
testing will be conducted by all designers over the course of 12/21/2021 to 1/3/2022

**Comment by lrchang2** [comment]
@miguel-hugo @joanna-pham @KaylenaNguyen @melissamurphy @ashleyxz @TracyLuu 
Please add update using this template (even if you have a pull request)

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures: "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 

1. ask for help at your next meeting
2. put a "Status: Help Wanted" label on your issue and pull request
3. put up a request for assistance on the #access-the-data channel.

**Comment by melissamurphy** [comment]
Progress: Desktop test completed.
No difficulties other than some Figma buttons being nonoperative to the participant.
I can work 2 additional hours on this issue this week if we need more tests, interpretation, or a presentation.

**Comment by ashleyxz** [comment]
Progress: Desktop usability test completed.
No difficulties or blockers.
Availability: conduct the mobile test this week.

**Comment by lrchang2** [comment]
reviewing results with team members at team meeting on 1/12/22

**Comment by lrchang2** [comment]
final edits that will be made to all high fi wireframes:

- enlarging desktop menu (words) 
- adding exit button for mobile menu 
- make carousel timing longer (both mobile and desktop) 
- make button look like it's been clicked on 

next steps are to work on spacing, ordering of information and copy.

**Comment by KaylenaNguyen** [comment]
I enlarged the desktop menu headings from 20 to 26.
The carousel timing is longer for both mobile and desktop - from 2000ms to 6000ms.
I added the same menu button to the mobile menu, when clicking it, it exits.
I also added an interaction for the about heading on the mobile menu. When you click About it will now go to the about page.
If I have time, I'll work on making the buttons look like it's been clicked.
We can also discuss spacing and ordering of information when we get there.

Everything seems to be good! Let me know.

**Comment by lrchang2** [comment]
edit: make header sticky

**Comment by KaylenaNguyen** [comment]
I tried to find different ways to get this done. I tried creating a layer over the header to be able to create the fixed position for the header. However, it's not possible to do so when it's on an auto-layout. I'm going to share a link that can better explain what the issue is. Figma doesn't allow for constraints on an auto-layout.

https://forum.figma.com/t/how-can-i-get-fix-position-when-scrolling-option-under-constraints-section/3782/5

Instead, I created the same pages without the auto-layout. I placed it under the most recent frames in Figma.
I thought this would be a good way for developers to see what we want done. This doesn't have to be permanent, but I thought it would be the best way for now. Let me know what you think. Or if you have any other ideas.

https://www.figma.com/file/nS5IqnutUT2FuPwwJObHaY/Access-the-Data-Main-Figma?node-id=2%3A16
